src/resources/postcss/tribe-events-single-blocks-reskin.pcss

```diff
@@ -49,3 +49,182 @@
+			flex-basis: 100%;
+			width: 100%;
+
+			@media screen and (min-width: 768px) {
+				flex-basis: 49%;
+				width: 49%;
+			}
+		}
+	}
+
+	.tribe-block__venue__meta {
+		margin-top: 24px;
+
+		@media screen and (min-width: 768px) {
+			margin-top: 0;
+		}
+
+		.tribe-block__venue__name {
+
+			h3 {
+				font-size: 1rem;
+				font-weight: normal;
+				letter-spacing: normal;
+				line-height: 1.64;
+			}
+		}
+
+		.tribe-block__venue__address {
+			margin-bottom: 0;
+
+			a {
+				color: var(--tec-color-link-accent);
+				margin-top: 17px;
+			}
+
+			.tribe-region.tribe-events-abbr {
+				text-decoration: none;
+			}
+		}
+
+		.tribe-block__venue__phone {
+			margin-bottom: 0;
+		}
+
+		.tribe-block__venue__address,
+		.tribe-block__venue__phone,
+		.tribe-block__venue__website {
+			color: #141827;
+			font-weight: normal;
+			letter-spacing: normal;
+			line-height: 1.64;
+		}
+	}
+}
+
+/* Theme specific overrides */
+
+/* Twenty-twenty-one theme */
+.tribe-theme-twentytwentyone .tribe-block__venue .tribe-block__venue__meta .tribe-block__venue__website a {
+	color: var(--tec-color-link-accent);
+}
+
+/* Twenty-twenty theme */
+.tribe-theme-twentytwenty {
+	.tribe-block__venue {
+		.tribe-block__venue__meta {
+			.tribe-block__venue__name h3,
+			.tribe-block__venue__address,
+			.tribe-block__venue__address a,
+			.tribe-block__venue__phone,
+			.tribe-block__venue__website {
+				font-size: 1.65rem;
+			}
+
+			.tribe-block__venue__website a {
+				color: var(--tec-color-link-accent);
+			}
+		}
+	}
+}
+
+/* Twenty Seventeen */
+.tribe-theme-twentyseventeen {
+	.tribe-block__venue {
+		.tribe-block__venue__meta {
+			.tribe-block__venue__name h3,
+			.tribe-block__venue__website a {
+				color: var(--tec-color-link-accent);
+			}
+		}
+	}
+}
+
+/* Divi */
+.tribe-theme-divi {
+	.tribe-block__venue {
+		.tribe-block__venue__meta {
+			.tribe-block__venue__name h3,
+			.tribe-block__venue__address,
+			.tribe-block__venue__phone,
+			.tribe-block__venue__website {
+				padding-bottom: 0;
+			}
+
+			.tribe-block__venue__website a {
+				color: var(--tec-color-link-accent);
+			}
+		}
+	}
+}
+
+/* Genesis */
+.tribe-theme-genesis {
+	.tribe-block__venue {
+		.tribe-block__venue__meta {
+			.tribe-block__venue__name h3,
+			.tribe-block__venue__address,
+			.tribe-block__venue__phone,
+			.tribe-block__venue__website {
+				font-size: 1.65rem;
+			}
+
+			.tribe-block__venue__website a {
+				color: var(--tec-color-link-accent);
+			}
+		}
+	}
+}
+
+/* Avada */
+.tribe-theme-avada {
+	.tribe-block__venue {
+		.tribe-block__venue__meta {
+
+			.tribe-block__venue__name {
+				.fusion-responsive-typography-calculated {
+					font-size: 1.2rem;
+					font-weight: normal;
+					letter-spacing: normal;
+				}
+			}
+
+			.tribe-block__venue__address a,
+			.tribe-block__venue__website a {
+				color: var(--tec-color-link-accent);
+			}
+		}
+	}
+}
+
+/* Enfold */
+.tribe-theme-enfold {
+	.tribe-block__venue {
+		.tribe-block__venue__meta {
```

Comment:
no need to unnecessarily nest here

Reply:
I added nesting to this because down the line we need to combine the theme overrides for each block. You can see this in action on this [PR](https://github.com/the-events-calendar/the-events-calendar/pull/3611).